microdnf.spec

```diff
@@ -1,7 +1,7 @@
 %global libdnf_version 0.43.1
```

Comment:
This version probably needs to be bumped.

Reply:
@Conan-Kudo  Version 0.43.1 is a topical version of libdnf. 

But I think that there is a bug in "context" part of libdnf. There is probably a missing test of timestamp of .repo file. If the .repo file is modified, then the repository's metadata cache should be invalidated regardless of age of the cache.
I'll look at it later.